schema/va-spec/core-im/core-im-source.yaml

```diff
@@ -331,7 +331,7 @@ $defs:
       generated. StudyResult objects provide a flexible and useful way to summarize a subset of data items
       from a larger study that are cited as evidence in curation workflows that generate higher order
       knowledge about a particular variant or other entity.
-    heritableProperties:
+    properties:
```

Comment:
@mbrush Is `StudyResult` a concrete class? I agree with `EvidenceLine` (below) being a concrete class, but I'm not sure anyone will be using `StudyResult` as-is without profiling it. If so, please explain.  I also am not sure about `Proposition` being considered a concrete class. I suppose `StudyGroup` is generic enough as-is, so I agree with that.